README.md

```diff
@@ -6,16 +6,18 @@ This is the open-source portion of the back-end, website scraping software that
 
 ## Setup
 
-1.  Download chromium version 843427 locally.
```

Comment:
I couldn't figure out how to easily download a specific older Chromium version, so I downloaded the most recent version and it's been working fine.